scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_view.py

```diff
@@ -35,6 +35,11 @@ def __init__(self, parent=None):
 
         self.disable_simul_fit_options()
 
+        if is_frequency_domain:
+            self.hide_simultaneous_fit_options()
+            self.fit_options_table.hideRow(3)
```

Comment:
what are these magic numbers?

Reply:
These are refencing the names of the table displaying information. I've changed them to named constants to make this clearer.